.gitignore

```diff
@@ -1,5 +1,8 @@
 /.gradle/
 /.idea/
+/plugin/out/
+/target/
+
```

Comment:
Is the blank line here now intended? Because `build/` below is also a directory, it's just not limited to match in the root.

Reply:
Removed the blank line.